Purchases/Networking/Operations/CreateAliasOperation.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CreateAliasOperation.swift
+//
+//  Created by Joshua Liebowitz on 11/18/21.
+
+import Foundation
+
+class CreateAliasOperation: CacheableNetworkOperation {
+
+    private let aliasCallbackCache: CallbackCache<AliasCallback>
+    private let createAliasResponseHandler: PostAttributionDataResponseHandler
```

Comment:
should `PostAttributionDataResponseHandler` maybe be renamed to something more generic?

Reply:
Oh, good find, yeah you're right!

Reply:
Updated!